src/Sulu/Bundle/MediaBundle/Media/FormatLoader/XmlFormatLoader.php

```diff
@@ -92,7 +92,13 @@ private function parseXml($file)
                     $parameters = array();
                     $parameterNodes = $this->xpath->query('x:parameters/x:parameter', $commandNode);
                     foreach ($parameterNodes as $parameterNode) {
-                        $parameters[$parameterNode->attributes->getNamedItem('name')->nodeValue] = $parameterNode->nodeValue;
+                        $value = $parameterNode->nodeValue;
+                        if ($value === 'true') {
+                            $value = true;
+                        } elseif ($value === 'false') {
+                            $value = false;
+                        }
+                        $parameters[$parameterNode->attributes->getNamedItem('name')->nodeValue] = $value;
```

Comment:
this is for all parameters.
if its not 'true' or 'false' the value whatever it is is set as parameter value 
https://github.com/sulu-io/sulu/blob/bugfix/image-keep-resolution/src/Sulu/Bundle/MediaBundle/Media/FormatLoader/XmlFormatLoader.php#L95